test/services/GooglePlacesService.test.ts

```diff
@@ -52,7 +52,7 @@ describe('Unit tests for GooglePlacesService', function () {
         expect(result).toHaveProperty('name', 'RIB & CHICKEN');
         expect(result).toHaveProperty('opening_hours');
         expect(result).toHaveProperty('photos');
-        expect(result).toHaveProperty('rating', 4.5);
```

Comment:
I removed these numerical values in case we add CI later for our tests- these numbers change pretty often

Reply:
Good catch!